build/app_template_files/__NativeSwiftTemplateAppName__/Podfile

```diff
@@ -2,10 +2,21 @@
 # platform :ios, '8.0'
 
 target '__NativeSwiftTemplateAppName__' do
+source 'https://github.com/CocoaPods/Specs.git'
+source 'https://github.com/wmathurin/SalesforceMobileSDK-iOS-Specs.git'
```

Comment:
The repo for our specs (for now on wmathurin, but would move to forcedotcom if we go forward)